src/types.ts

```diff
@@ -74,13 +74,15 @@ export interface FormatterProps<TRow, TSummaryRow = unknown> {
   row: TRow;
   isCellSelected: boolean;
```

Comment:
May be we no longer need `isCellSelected` prop

Reply:
If we don't use it at all then let's remove it.
We can add it back later if we find a use for it.